parser.h

```diff
@@ -126,6 +173,7 @@ class Boolean : public Literal
 
 class Nil : public Literal
 {
+    // TODO: should we add type here?
```

Comment:
Yeah, just make constructor with no arguments which will assign the value of execType

Reply:
OK, I added this.